pkgs/by-name/te/textlint-rule-diacritics/package.nix

```diff
@@ -21,6 +21,10 @@ buildNpmPackage rec {
 
   dontNpmBuild = true;
 
+  postInstall = ''
+    find "$out/lib/node_modules" -xtype l -delete
```

Comment:
I will set `dontCheckForBrokenSymlinks` here instead of removing broken symlinks

https://github.com/NixOS/nixpkgs/pull/461250#discussion_r2525658329